cmd/acr/cssc.go

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package main
+
+import (
+	"context"
+	"fmt"
+
+	orasauth "github.com/Azure/acr-cli/auth/oras"
+	"github.com/Azure/acr-cli/internal/cssc"
+
+	"github.com/Azure/acr-cli/cmd/api"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+const (
+	newCsscCmdLongMessage  = `acr cssc: CSSC (Container Secure Supply Chain) operations for a registry. Use subcommands for more specific operations.`
+	newPatchCmdLongMessage = `acr cssc patch: List all repositories and tags that match the filter.
```

Comment:
Add the preview label to these commands so that its not indicated as GA. 
For e.g. https://github.com/oras-project/oras/blob/main/cmd/oras/root/discover.go#L49

Reply:
Addressed this in the latest commit.